src/iconsManifest/supportedExtensions.ts

```diff
@@ -4193,7 +4193,7 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'terraform',
-      extensions: ['tfstate'],
+      extensions: ['tfstate', 'tfvars', 'tf', 'tf.json'],
```

Comment:
Note that `tf` should be supported by the language id: https://github.com/hashicorp/vscode-terraform/blob/main/package.json#L62-L68 

Anyway, we can keep it like this in this occasion.